apps/mark/frontend/src/pages/admin_screen.test.tsx

```diff
@@ -23,7 +24,8 @@ import {
 let apiMock: ApiMock;
 
 beforeEach(() => {
-  jest.useFakeTimers().setSystemTime(new Date('2020-10-31T00:00:00.000'));
+  vi.useRealTimers();
+  // vi.useFakeTimers().setSystemTime(new Date('2020-10-31T00:00:00.000'));
```

Comment:
Remove comment.

Also, why do we need `vi.useRealTimers` here? I don't see anywhere that we're setting fake timers as a default for all tests.

Reply:
Remove.